packages/app/src/components/uiBlocks/LabeledInput.vue

```diff
@@ -0,0 +1,21 @@
+<template>
```

Comment:
Where was the need for a labeled input, when you could use our texfield and use the label option? What were the problems you encountered?

Reply:
I removed the label option from our `TextField` and `Dropdown` and moved it into a separate component to unify the label code a bit.